guide/git_flow/git-flow.md

```diff
@@ -14,6 +14,11 @@ set of steps outlined in the following document:
 Git workflow
 :::
 
+1. SETUP: Download an online repository using `git clone [url repo]` or start your own local Git repository using `git init`.
+2. BRANCH: Create your safe environment using `git branch [branch-name]`. This isolates your local work and allows testing additional features. 
+3. STAGE & COMMIT: Stage your recent work using `git add [modified-file]` and save it with `git commit -m "[comment]"`.
```

Comment:
```suggestion
3. STAGE & COMMIT: Stage your recent work using `git add [modified-file]` and save it with `git commit -s -m "[comment]"`.
```

I'll explain this next week. `-s`/`--signoff` will be necessary in our organization to pass the DCO automatic check in the CI pipeline.

Reply:
Thanks for your comment! I have added this improvement to the page.